database-seeds/scripts/rerunners/maimaidx/merge-options.ts

```diff
@@ -69,8 +70,7 @@ interface MusicXML {
 }
 
 function calculateLevel(data: Pick<NotesData, "level" | "levelDecimal">) {
-	// TODO: Change levelDecimal condition to 6 when updating to BUDDiES+
-	return `${data.level}${data.levelDecimal >= 7 && data.level >= 7 ? "+" : ""}`;
+	return `${data.level}${data.levelDecimal >= 6 && data.level >= 6 ? "+" : ""}`;
```

Comment:
```suggestion
	return `${data.level}${data.levelDecimal >= 6 && data.level >= 7 ? "+" : ""}`;
```

While the condition for `+` levels have been changed from `.7` to `.6`, the lowest level with a `+` is still 7. There is no level 6+.